scripts/test/generate_lld_tests.py

```diff
@@ -1,5 +1,4 @@
 #!/usr/bin/env python
-
```

Comment:
Is this a flake8 thing? I personally prefer the semantically-meaningful comment (the shebang) to be visually separated from the copyright boilerplate, though this is the sort of thing I can't really bring myself to get worked up about.

Reply:
I've standardized on using a live to separate but with a leading `#`.  This mimiks what wabt does